src/openforms/forms/tests/test_migrations.py

```diff
@@ -1102,3 +1102,61 @@ def test_the_custom_error_is_added(self):
                 },
             },
         )
+
+
+class TestAddDateComponentSettings(TestMigrations):
+    app = "forms"
+    migrate_from = "0092_more_time_custom_errors"
+    migrate_to = "0093_date_component_settings"
+
+    def setUpBeforeMigration(self, apps):
+        FormDefinition = apps.get_model("forms", "FormDefinition")
+        FormStep = apps.get_model("forms", "FormStep")
+        Form = apps.get_model("forms", "Form")
+
+        self.form_def = FormDefinition.objects.create(
+            name="Date",
+            slug="date",
+            configuration={
+                "components": [
+                    {
+                        "key": "dateComponent",
+                        "type": "date",
+                        "label": "Date component",
+                        "customOptions": {},
```

Comment:
let's add a component in this test that doesn't have this key at all to verify my comment above :)